packages/commonwealth/client/scripts/views/pages/create_community/cosmos_form.tsx

```diff
@@ -39,6 +50,21 @@ export const CosmosForm = () => {
 
   const navigate = useCommonNavigate();
 
+  const chainNameValidationFn = (
+    value: string
+  ): [ValidationStatus, string] | [] => {
+    const validChainNameRegex = /^[a-z0-9]+$/;
+
+    if (!validChainNameRegex.test(value)) {
+      return [
+        'failure',
+        'Must be lowercase, alphanumeric, and equal to Cosmos Chain Registry entry',
```

Comment:
So we don't actually validate the correctness of the chain id?

Reply:
We _could_, assuming they register their chain with the registry before starting a community with CW. We could query a registry API to make sure a chain "exists" in the registry.

Reply:
I think this might be the correct approach -- see how the create community page uses `await $.getJSON('https://chainid.network/chains.json');`, which we use to verify ETH chain names.

Reply:
So, validation on the front end or back end preferred? @jnaviask

Reply:
I think currently only admins can add new cosmos communities anyway, but we would eventually like to open it up, so I would say backend is ideal.

Reply:
OK. I started with front end validation, and now I have both. Is that overkill or good? @jnaviask

Reply:
I just realized we could make the chain_name input a dropdown instead of a text input. Since we can easily fetch them all. We could even eliminate chains that are already set up. Maybe a follow-up ticket?

Reply:
Just doing back-end validation for now.